platform_view_swift/test/widgets_test.dart

```diff
@@ -0,0 +1,11 @@
+// This is a basic Flutter widget test.
+// To perform an interaction with a widget in your test, use the WidgetTester utility that Flutter
+// provides. For example, you can send tap and scroll gestures. You can also use WidgetTester to
+// find child widgets in the widget tree, read text, and verify that the values of widget properties
+// are correct.
+
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('This test always passes', (WidgetTester tester) async {});
```

Comment:
:sad filip:

Let's have a fixit one day where we go through all of these tests of samples and actually write them. Just one smoke test per sample is all we need.

Reply:
Adding issues for this now.